classes/local/util/manager.php

```diff
@@ -154,6 +154,41 @@ protected function prepare_files_skeletons() {
                 $this->prepare_file_skeleton('db/upgradelib.php', 'php_internal_file', 'db_upgradelib');
             }
         }
+
+        if ($this->should_have('backup_moodle2')) {
+            $this->prepare_backup_moodle2();
+        }
+    }
+
+    /**
+     * Prepares the skeleton files for the 'backup_moodle2' feature.
+     */
+    public function prepare_backup_moodle2() {
+
+        // TODO: Lib.php will advertise the fact that the plugin supports backup.
```

Comment:
This is the place where I see a problem. We need to inject content into lib.php from multiple sources